sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.Metrics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal static class Statsbeat
+    {
+        private static readonly Meter s_myMeter = new("AttachStatsBeatMeter", "1.0");
+
+        private const string StatsBeat_ConnectionString = "<StatsBeat_ConnectionString>";
+
+        internal const int AttachStatsBeatInterval = 86400000;
+
+        static Statsbeat()
+        {
+            s_myMeter.CreateObservableGauge("AttachStatsBeat", () => GetAttachStatsBeat());
+
+            // Configure for attach statsbeat which has collection
+            // schedule of 24 hrs == 86400000 milliseconds.
+            Sdk.CreateMeterProviderBuilder()
+            .AddMeter("AttachStatsBeatMeter")
+            .AddAzureMonitorMetricExporter(o =>
```

Comment:
no need of using this extension method. can we not add the reader ourself by instantiating AzureMonitormetricExporter?

Reply:
updated